pkg/controller/flow/flow.go

```diff
@@ -170,6 +186,9 @@ func NewController(
 		},
 	})
 
+	// TODO: const for knative-system
+	controller.controllerConfigMapWatcher = servingconfigmaps.NewDefaultWatcher(kubeclientset, "knative-system")
```

Comment:
should this be knative-eventing?

Reply:
also think there's a way to get the eventing namespace... time passes... yeah, pkg/names.go

Reply:
use system.Namespace for now.